src/zh/boylove/src/eu/kanade/tachiyomi/extension/zh/boylove/BoyLoveFilters.kt

```diff
@@ -17,20 +17,30 @@ import eu.kanade.tachiyomi.source.model.FilterList
 internal fun parseFilters(page: Int, filters: FilterList): String {
     var status = '2'
     var type = '0'
+    var region = "0"
     var genre = "0"
     var sort = '1'
     var vip = '2'
     for (filter in filters) {
         when (filter) {
             is StatusFilter -> status = STATUS_KEYS[filter.state]
             is TypeFilter -> type = TYPE_KEYS[filter.state]
+            is RegionFilter -> region = REGION_KEYS[filter.state]
             is GenreFilter -> if (filter.state > 0) genre = filter.values[filter.state]
             is SortFilter -> sort = SORT_KEYS[filter.state]
             is VipFilter -> vip = VIP_KEYS[filter.state]
             else -> {}
         }
     }
-    return "1-$genre-$status-$sort-$page-$type-1-$vip"
+
+    var tags = when {
```

Comment:
```suggestion
    val tags = when {
```

Not sure why that's a typo.